content/en/hosting/medic/_index.md

```diff
@@ -4,4 +4,8 @@ linkTitle: "At Medic"
 weight: 300
 ---
 
-{{< subpages >}}
\ No newline at end of file
+{{< hextra/hero-subtitle >}}
```

Comment:
According to [this PR](https://github.com/medic/cht-docs/pull/1886), we don't need anymore to add the `hextra/hero-subtitle` for descriptions, as they are automatically generated from the `description` header field.

Reply:
Thanks Andra! Fixed in cf2200d5